File: Practice/programmers_079.py
from itertools import combinations
from collections import Counter

def solution(orders, course):
    answer = []
    arr = []
    for cnt in course:
        brr = []
        for order in orders:
            for comb in combinations(order, cnt):
                res = ''.join(sorted(comb))
                brr.append(res)
        sort_counter = Counter(brr).most_common()
        answer += [menu for menu, cnt in sort_counter if cnt > 1 and cnt == sort_counter[0][1]]
    return sorted(answer)

File: JavaScript/programmers_079.js
// LV.2 위장

function solution(clothes) {
  var answer = 1;
  let section = {};
  for (let i = 0; i < clothes.length; i++) {
    if (!section[clothes[i][1]]) section[clothes[i][1]] = 1;
    else section[clothes[i][1]] += 1;
  }

  for (let value of Object.values(section)) {
    answer *= value + 1;
  }
  return answer - 1;
}
